### PR TITLE
Add possibility to list bandwidths

### PR DESCRIPTION
**Pull request by outcatcher**
Implement EIP bandwidth list

Doc: https://docs.otc.t-systems.com/api/eip/eip_apiBandwidth_0002.html

### Acceptance tests:

```
=== RUN   TestBandwidthsList
    bandwidths_test.go:86: Attempting to create eip/bandwidth: testacc-fW1tL9LI
    bandwidths_test.go:104: Waiting for eip 818eeb6a-b9b4-49b5-ba78-dd8bb9ab56b5 to be active
    bandwidths_test.go:113: Created eip/bandwidth: testacc-fW1tL9LI
    bandwidths_test.go:76: Bandwith with ID 5bad82b9-09bf-4167-8a5f-c433794f69a0 is in bandwidth list
    bandwidths_test.go:119: Attempting to delete eip/bandwidth: 818eeb6a-b9b4-49b5-ba78-dd8bb9ab56b5
    bandwidths_test.go:127: Waitting for eip 818eeb6a-b9b4-49b5-ba78-dd8bb9ab56b5 to be deleted
    bandwidths_test.go:132: Deleted eip/bandwidth: 818eeb6a-b9b4-49b5-ba78-dd8bb9ab56b5
--- PASS: TestBandwidthsList (9.57s)
PASS
```

### CODE REVIEW

File: openstack/networking/v1/bandwidths/results.go [hunk 1]
```diff
@@ -68,3 +68,14 @@ func (r UpdateResult) Extract() (BandWidth, error) {
...
 	return bw, err
 }
+
+// ListResult is a struct which contains the result of list method
+type ListResult struct {
+	golangsdk.Result
+}
+
+func (r ListResult) Extract() ([]BandWidth, error) {
+	var s []BandWidth
```
Comment (anton-sidelnikov) [inline openstack/networking/v1/bandwidths/results.go:78]: maybe call it bws not s

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]
cmt

**Review by lego963 — CHANGES_REQUESTED** [review]
add test pls

**Review by anton-sidelnikov — APPROVED** [review]

**Review by lego963 — APPROVED** [review]

File: acceptance/openstack/networking/v1/bandwidths_test.go [hunk 2]
```diff
@@ -47,6 +47,39 @@ func TestBandwidthsUpdate(t *testing.T) {
...
+	existingBandwidths, err := bandwidths.List(client, bandwidths.ListOpts{
+		ShareType: "PER",
+	}).Extract()
+	if err != nil {
+		t.Fatalf("Unable to list bandwidths: %s", err)
+	}
+
+	for _, b := range existingBandwidths {
+		if b.ID == eip.BandwidthID {
+			t.Logf("Bandwith with ID %s is in bandwidth list", b.ID)
```
Comment (lego963) [inline acceptance/openstack/networking/v1/bandwidths_test.go:76]: add here `tools.PrintResource(b)` pls

**Review by anton-sidelnikov — APPROVED** [review]

**Review by lego963 — APPROVED** [review]

**Pull request merged**